github_jira_sync_app/main.py

```diff
@@ -239,6 +239,25 @@ async def bot(request: Request, payload: dict = Body(...)):
         "description": issue_description,
         "issuetype": {"name": issue_type},
     }
+
+    # allow customization of JIRA issue summary field
+    # by using this field, use can format the JIRA summary with the values
+    # from the GH issue
+    # Examples:
+    # to use fixed title "github issue": "github issue"
+    # to use GH issue title: "{issue.title}"
+    # to add prefix "GitHub" : "GitHub {issue.title}"
+    # to add user in the title: "[issue.user.login] {issue.title}"
+    summary_str = settings.get("summary", "")
+    if isinstance(summary_str, str) and summary_str != "":
+        try:
+            summary_str = settings["summary"].format(issue=issue)
```

Comment:
that is very implicit logic. So, you expect that in the settings file user will supply some preformatted string with variably `issue`.

please make it more explicit in the comment and in the text in Readme

Reply:
@beliaev-maksim  Thanks ! I already provided some comments, do you think that it is not explicit enough ? anyway, i tried to improve the comments and the comment in the README in a new commit.

Reply:
for me it is not enough, it takes a bit to understand what and how we do here

Reply:
@beliaev-maksim  I did an other try to improve the README, please check. btw, if you have any recomendation to make it better, feel free to let me know.